BorrowBookControl.java

```diff
@@ -0,0 +1,116 @@
+import java.util.ArrayList;
+import java.util.List;
+
+public class BorrowBookControl {
+
+	private BorrowBookUI ui;
```

Comment:
DEFECT Custom: ui should always be UI (As discussed), give this member a more meaningful name. bookUI is fine.

Reply:
done